.devcontainer/features/languages/nodejs/install.sh

```
@@ -219,7 +251,7 @@ if [ -n "$GLOBAL_PACKAGES" ]; then
     for pkg in "${PKGS[@]}"; do
         pkg=$(echo "$pkg" | xargs)  # trim whitespace
         if [ -n "$pkg" ] && ! command -v "$pkg" &>/dev/null; then
-            npm install -g "$pkg" 2>/dev/null && log_success "Installed $pkg" || log_warning "Failed to install $pkg"
+            npm_install_global "$pkg" && log_success "Installed $pkg" || log_warning "Failed to install $pkg (non-critical)"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)